src/test/mir-opt/const_allocation/rustc.main.ConstProp.after.mir

```diff
@@ -4,85 +4,65 @@
                                          // ty::Const
                                          // + ty: &&[(std::option::Option<i32>, &[&str])]
                                          // + val: Value(Scalar(alloc0+0))
                                          // mir::Constant
-                                         // + span: $DIR/const_allocation.rs:6:5: 6:8
+                                         // + span: $DIR/const_allocation.rs:8:5: 8:8
                                          // + literal: Const { ty: &&[(std::option::Option<i32>, &[&str])], val: Value(Scalar(alloc0+0)) }
-        _1 = (*_2);                      // bb0[3]: scope 0 at $DIR/const_allocation.rs:6:5: 6:8
-        StorageDead(_2);                 // bb0[4]: scope 0 at $DIR/const_allocation.rs:6:8: 6:9
-        StorageDead(_1);                 // bb0[5]: scope 0 at $DIR/const_allocation.rs:6:8: 6:9
-        _0 = ();                         // bb0[6]: scope 0 at $DIR/const_allocation.rs:5:11: 7:2
-        return;                          // bb0[7]: scope 0 at $DIR/const_allocation.rs:7:2: 7:2
+        _1 = (*_2);                      // bb0[3]: scope 0 at $DIR/const_allocation.rs:8:5: 8:8
+        StorageDead(_2);                 // bb0[4]: scope 0 at $DIR/const_allocation.rs:8:8: 8:9
+        StorageDead(_1);                 // bb0[5]: scope 0 at $DIR/const_allocation.rs:8:8: 8:9
+        _0 = ();                         // bb0[6]: scope 0 at $DIR/const_allocation.rs:7:11: 9:2
+        return;                          // bb0[7]: scope 0 at $DIR/const_allocation.rs:9:2: 9:2
     }
 }
 
-alloc0 {
-    size: 16
-    alignment: 8
-    static const_allocation[317d]::FOO[0]
-    memory: ├──────alloc17+0──────┤ 03 00 00 00 00 00 00 00 
+alloc0 (const_allocation[317d]::FOO[0]) size: 16, align: 8 {
+    ├──────alloc17+0──────┤ 03 00 00 00 00 00 00 00 
 }
 
-alloc17 {
-    size: 72
-    alignment: 8
-    memory: 00 00 00 00 __ __ __ __ ├──────alloc4+0───────┤ 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __ ├──────alloc8+0───────┤ 02 00 00 00 00 00 00 00 01 00 00 00 2a 00 00 00 ├──────alloc13+0──────┤ 03 00 00 00 00 00 00 00 
+alloc17 size: 72, align: 8 {
+    0x00 │ 00 00 00 00 __ __ __ __ ├──────alloc4+0───────┤ 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __ 
+    0x20 │ ├──────alloc8+0───────┤ 02 00 00 00 00 00 00 00 01 00 00 00 2a 00 00 00 ├──────alloc13+0──────┤ 
+    0x40 │ 03 00 00 00 00 00 00 00 
 }
 
-alloc13 {
-    size: 48
-    alignment: 8
-    memory: ├──────alloc12+0──────┤ 03 00 00 00 00 00 00 00 ├──────alloc14+0──────┤ 03 00 00 00 00 00 00 00 ├──────alloc15+0──────┤ 04 00 00 00 00 00 00 00 
+alloc13 size: 48, align: 8 {
+    0x00 │ ├──────alloc12+0──────┤ 03 00 00 00 00 00 00 00 ├──────alloc14+0──────┤ 03 00 00 00 00 00 00 00 
+    0x20 │ ├──────alloc15+0──────┤ 04 00 00 00 00 00 00 00 
 }
 
-alloc15 {
-    size: 4
-    alignment: 1
-    memory: 6d c3 b6 70 
+alloc15 size: 4, align: 1 {
```

Comment:
Maybe the details could go in parens, i.e.:
```rust
alloc15 (size: 4, align: 1) {
    6d c3 b6 70 
}
```

Reply:
so `alloc15 (path::to::STATIC, size: 4, align: 1) {`?

Reply:
I'd propose `alloc15 (static: path::to::STATIC, size: 4, align: 1) {`